test/unit/org/apache/cassandra/cql3/validation/operations/CreateTest.java

```diff
@@ -688,12 +692,16 @@ public void testCreateTableWithCompression() throws Throwable
     @Test
     public void testUseUnreservedKeywordAsColumnName()
     {
-        List<String> names = Arrays.asList("access", "datacenters");
-        for (String colName : names)
+        Set<String> allKeywords = Arrays.stream(CqlLexer.class.getDeclaredFields())
+                                        .filter(f -> f.getName().startsWith("K_"))
+                                        .map(f -> f.getName().substring(2)) // remove the heading "K_"
+                                        .collect(Collectors.toSet());
+        Set<String> unreservedKeywords = Sets.difference(allKeywords, ReservedKeywords.reservedSet);
```

Comment:
Another approach would be to make `isReserved(String text)` `public` and to use it to filter the stream.
```

Set<String> unreservedKeywords = Arrays.stream(CqlLexer.class.getDeclaredFields())
                                                                  .filter(f -> f.getName().startsWith("K_"))
                                                                  .map(f -> f.getName().substring(2)) // remove the heading "K_"
                                                                  .filter(f -> !ReservedKeywords.isReserved(f))
                                                                 .collect(Collectors.toSet());
```
Making `isReserved(String text)` `public` allow us to expose less of the class internals but I do not really have a strong opinion on which solution we should use.